index.d.ts

```diff
@@ -165,10 +164,10 @@ declare namespace megajs {
     }
     interface FileOpts {
         api?: API;
-        key?: BufferString;
+        key: BufferString;
```

Comment:
Key is not required. Well, it should not, but now I see that I forgot to add tests to handle this case. The documentation at least says it's not required (it is in the security page).